plugins/module_utils/network/iosxr/config/bgp_global/bgp_global.py

```diff
@@ -0,0 +1,402 @@
+            'local_as',
+            'local',
+            'local_address',
+            'origin_as',
+            'remote_as',
+            'receive_buffer_size',
+            'send_buffer_size',
+            'session_open_mode',
+            'neighbor_shutdown',
+            'neighbor_shutdown_inheritance_disable',
+            'neighbor_tcp_mss',
+            'neighbor_tcp_mss_inheritance_disable',
+            'neighbor_timers_keepalive',
+            'update_source',
+            'neighbor_ttl_security_inheritance_disable',
+            'neighbor_ttl_security',
+            'neighbor_graceful_maintenance_set',
+            'neighbor_graceful_maintenance_activate',
+            'neighbor_graceful_maintenance_activate_inheritance_disable',
+            'neighbor_graceful_maintenance_as_prepends',
+            'neighbor_graceful_maintenance_local_preference_disable',
+            'neighbor_graceful_maintenance_local_preference',
+            'neighbor_graceful_maintenance_as_prepends_value',
+            'neighbor_capability_additional_paths_send',
+            'neighbor_capability_additional_paths_send_disable',
+            'neighbor_capability_additional_paths_rcv_disable',
+            'neighbor_capability_additional_paths_rcv',
+            'neighbor_capability_suppress_4_byte_AS',
+            'neighbor_capability_suppress_all',
+            'neighbor_capability_suppress_all_inheritance_disable',
+            'neighbor_log_message_in_value',
+            'neighbor_log_message_in_disable',
+            'neighbor_log_message_in_inheritance_disable',
+            'neighbor_log_message_out_value',
+            'neighbor_log_message_out_disable',
+            'neighbor_log_message_out_inheritance_disable',
+            'neighbor_update_in_filtering_attribute_filter_group',
+            'neighbor_update_in_filtering_logging_disable',
+            'neighbor_update_in_filtering_message_buffers',
+           ]
+
+        want_nbr = want.get("neighbors", {})
+        have_nbr = have.get("neighbors", {})
+        for name, entry in iteritems(want_nbr):
+            have = have_nbr.pop(name, {})
+            begin = len(self.commands)
+            self.compare(parsers=neighbor_parsers, want=entry, have=have)
+            neighbor_address = entry.get("neighbor", "")
+            if len(self.commands) != begin:
+                self.commands.insert(
+                    begin, self._tmplt.render({"neighbor": neighbor_address}, "neighbor", False)
+                )
+        for name, entry in iteritems(have_nbr):
+            if self._check_af("neighbor", name):
+                self._module.fail_json(
+                    msg="Neighbor {0} has address-family configurations. "
+                        "Please use the iosxr_bgp_neighbor_address_family module to remove those first.".format(
+                        name
+                    )
+                )
+            else:
+                self.addcmd(entry, "neighbor", True)
```

Comment:
Will the other attributes in have->neighbor , which are not part of want-neighbor , be negated?